aodntools/timeseries_products/velocity_aggregated_timeseries.py

```diff
@@ -225,7 +225,7 @@ def aggregate_velocity(files_to_agg, site_code, base_path):
                 WCURqc[start:end] = np.full(varlen_list[index], np.nan)
             ##calculate depth
             if 'HEIGHT_ABOVE_SENSOR' in nc.dims:
-                DEPTH[start:end] = (nc.DEPTH + nc.HEIGHT_ABOVE_SENSOR.T).values.flatten()
```

Comment:
Since this was undetected last time, I think a validation would be worth here (Just in case).

Reply:
check added to check_file function